### PR TITLE
feat(notifications): Manage audio notifications

### PR DESCRIPTION
**Pull request by horymury**
- manage recording audio notifications through config

@saghul @damencho

### CODE REVIEW

**Comment by horymury** [comment]
@saghul applied the approach we discussed. this is not WIP any more.

cc: @damencho

File: react/features/recording/middleware.js [hunk 1]
```diff
@@ -44,6 +44,8 @@ import {
     RECORDING_ON_SOUND_FILE
 } from './sounds';
 
+declare var config: Object;
```
Comment (saghul) [inline react/features/recording/middleware.js:47]: Please don't use the config global, access the state on the config feature.

Comment (horymury) [reply react/features/recording/middleware.js]: done

**Comment by saghul** [comment]
Nice work @horymury !

**Pull request merged**